cassandra/cluster.py

```diff
@@ -3771,8 +3772,11 @@ def _execute_after_prepare(self, host, connection, pool, response):
                 if self.prepared_statement:
                     # result metadata is the only thing that could have
                     # changed from an alter
-                    _, _, _, result_metadata = response.results
-                    self.prepared_statement.result_metadata = result_metadata
+                    (_, _, _,
+                     self.prepared_statement.result_metadata,
+                     new_metadata_id) = response.results
```

Comment:
That is becoming a bit messy. We should probably create a follow-up ticket to create a better structure for response results.

Reply:
We've already got a ticket for it, actually: [PYTHON-601](https://datastax-oss.atlassian.net/browse/PYTHON-601)